server/config/config.go

```diff
@@ -446,14 +492,14 @@ func (man Manager) addConfigs() {
 		"(DEPRECATED: Use filesystem.enable_log_rotation) Enable automatic rotation for osquery log files")
 	man.addConfigInt("osquery.max_jitter_percent", 10,
 		"Maximum percentage of the interval to add as jitter")
-	man.addConfigBool("osquery.enable_async_host_processing", false,
```

Comment:
So this kind of change is backwards compatible because it only changes how the value is parsed, but false/0/no would parse the same, right?

Reply:
Yeah, I tested with various yaml bool encodings (env vars and flags are strings to begin with anyway, so that's less of a concern although tested too) to make sure that they were all still parsed properly.